src/components/Wrap/index.tsx

```diff
@@ -1,25 +1,32 @@
 /** @jsx jsx */
 import * as React from "react";
-import { jsx, css } from "@emotion/core";
+import { jsx } from "@emotion/core";
+import { mq } from "../../../lib/utils";
 import { Theme } from "../../types/global";
 
-type MaxWidths = 1 | 2 | 3;
+type MaxWidths = 1 | 2;
 
 interface Props {
   as?: string;
   size?: MaxWidths;
 }
 
 const Wrap: React.FC<Props> = ({ as: W = "div", size = 1, ...props }) => {
-  const styles = ({ ...theme }: Theme) =>
-    css({
-      marginLeft: "auto",
-      marginRight: "auto",
-      maxWidth: size && theme.maxWidths[size - 1],
-      width: "100%",
-    });
-
-  return <W css={styles} {...props} />;
+  return (
+    <W
```

Comment:
I know this was added previously, but reading this from a specification standpoint, I can't seem to draw significance from the parameter `W`. Perhaps `El` would better convey its purpose?

Reply:
Yeah `El` makes a lot more sense

Reply:
This will be fixed with #33